src/components/Formik/Input/Input.tsx

```diff
@@ -2,6 +2,7 @@ import {useField} from "formik";
 import React from "react";
 import '../Input/input.scss'
 
+
 interface Props {
```

Comment:
Masz tendencje do wrzucania pustych enterów do plików 😄 Jak widzisz na commit, że w pliku nie ma zmian poza enterem to cofnij ten enter zanim go wrzucisz - mniej do przeglądania na CR 😃

Reply:
zrobione

Reply:
lub używaj wtyczki eslint i prettier, formatują kod automatycznie :)